packages/headless/src/controllers/commerce/context/cart/headless-cart.ts

```diff
@@ -90,10 +90,9 @@ export interface Cart extends Controller {
   /**
    * Emits an `ec.purchase` analytics event and then empties the cart without emitting any additional events.
    *
-   * @param transactionId - The transaction ID.
-   * @param transactionRevenue - The total revenue from the transaction, including taxes, shipping, and discounts.
+   * @param transaction - The object with the id and the total revenue from the transaction, including taxes, shipping, and discounts.
    */
-  purchase(transactionId: string, transactionRevenue: number): void;
+  purchase(transaction: {id: string; revenue: number}): void;
```

Comment:
Do you think the transaction interface will be used in other circumstances than emitting an event? I'm asking, because I wonder if it should be added to the @coveo/relay-event-types package